jira_google_calendar/deadline_to_event/README.md

```diff
@@ -9,8 +9,77 @@ This program is a real-life example workflow integrating Jira and Google Calenda
 
 ## How It Works
 
-- **Trigger**: The creation of a Jira issue in the designated project (specified in the [`autokitteh.yaml`](./autokitteh.yaml) file).
-- **Result**: A new event is created in the user's Google Calendar containing information from the Jira issue (e.g., `duedate`).
+1. **Trigger**: The workflow is triggered by the creation of a new Jira issue.
+2. **Create Calendar Event**: The program creates a Google Calendar event based on the Jira issue details, including the due date and description.
+3. **Notify**: The event is created in the Google Calendar, and attendees are notified.
```

Comment:
Would be nice to see a screenshot of both Jira and Calendar (not in high priority)

Reply:
I had trouble initializing connections and so wasn't able to run the project. Not sure what's going on.